packages/browser-agent-core/features/metrics/instrument/workers-helper.js

```diff
@@ -0,0 +1,82 @@
+export default {workersApiIsSupported, insertSupportMetrics};
```

Comment:
Do you mean importing individual (list) rather than a default object?

Reply:
when webpack sees `export default ...`  it will always have to include all the contents of the exports in each chunk that references that default export, because it is all imported even if not all the methods are used.  

Obviously not a huge deal here, but just wanted to say that was the pattern we were following.

It would be a syntax change from
`export default {workersApiIsSupported, insertSupportMetrics}`
to 
`export const workersApiIsSupported = ...`
`export function insertSupportMetrics() {...}`